src/main/java/com/sap/sgs/phosphor/fosstars/data/github/UsesGithubForDevelopment.java

```diff
@@ -118,26 +114,43 @@ static boolean usesGitHubForDevelopment(GHRepository repository, double threshol
       return true;
     }
 
+    // if pull requests are enabled for the repository, then it looks like GitHub is used for dev
+    if (enabledPullRequests(repository)) {
+      return true;
+    }
+
     int points = CHECKS.stream()
         .map(check -> check.test(repository) ? 1 : 0)
         .reduce(0, Integer::sum);
 
-    return (double) points / CHECKS.size() >= threshold;
+    return Double.compare((double) points / CHECKS.size(), threshold) >= 0;
+  }
+
+  /**
+   * Check if pull requests are enabled in a repository.
+   *
+   * @param repository The repository to be checked.
+   * @return True if pull requests are enabled, false otherwise.
+   */
+  private static boolean enabledPullRequests(GHRepository repository) {
+    // if one of the three options for pull requests is enabled, then it looks like
+    // that the project uses pull requests
+    return repository.isAllowMergeCommit()
+        || repository.isAllowRebaseMerge()
+        || repository.isAllowSquashMerge();
   }
 
   /**
    * Check if a string is not a URL to GitHub.
    * 
    * @param input The string to be checked.
    * @return True if the string is not a URL to GitHub, false otherwise.
-   * @throws IOException If something goes wrong.
    */
-  private static boolean notGitHubUrl(String input) throws IOException {
+  static boolean notGitHubUrl(String input) {
```

Comment:
Just a question, any specific reason for checking the negativity here? I mean it could have been isGitHubUrl() right?

Reply:
It could be `isGitHubUrl()`. One of the reasons may be that we are interested in non-GitHub URLs. Maybe @sourabhsparkala can give another reason since he wrote this originally.